### PR TITLE
LPS-149334,LPS-149336,LPS-149335

### PR DESCRIPTION
**Pull request by rebsilva**
Related Issues: 

- https://issues.liferay.com/browse/LPS-149334
- https://issues.liferay.com/browse/LPS-149336
- https://issues.liferay.com/browse/LPS-149335

Hello Reviewer!

This PR has new features related to the story:
**As a power user, I want to limit the number of characters of text and long text fields** ([LPS-146889](https://issues.liferay.com/browse/LPS-146889))

More details can be found [here](https://liferay.atlassian.net/wiki/spaces/ENGFORMS/pages/1994490142/LPS-146889+-+As+a+power+user+I+want+to+limit+the+number+of+characters+of+text+and+long+text+fields) on our Confluence page.

If you have any questions feel free to ask, thank you for your time!

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -13483,6 +13484,9 @@ set-the-google-analytics-id-that-is-used-for-this-set-of-pages=Set the Google An
 set-the-google-maps-api-key-that-is-used-for-this-set-of-pages=Set the Google Maps API Key that is used for this set of pages.
 set-the-google-places-api-key-that-is-used-for-this-set-of-pages=Set the Google Places API Key that is used for this set of pages.
 set-the-key-for-the-bing-maps-api-integration=Set the key for the Bing Maps API integration. See https://msdn.microsoft.com/en-us/library/ff428642.aspx for more information.
+set-the-maximum-number-of-characters=Set the Maximum Number of Characters
+set-the-maximum-number-of-characters-accepted-this-value-cant-be-less-than-1-or-greater-than-280=Set the maximum number of characters accepted. This value cannot be less than 1 or greater than 280.
```
Comment (rodrigopaulino) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:13488]: Hey, @rebsilva could these two be only one with a parametrized max and min?

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 1 above)
Comment (rebsilva) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:13488]: Sure @rodrigopaulino! I will try to do that, thanks!

**Comment by liferay-forms** [comment]
Hey @rebsilva, can you please check if these failures are related to your changes? Thanks

**Comment by rebsilva** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hey @rebsilva, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @rebsilva, can you please fix the source formatter issues and resend? Thanks

**Comment by rebsilva** [comment]
ci:test:sf

**Comment by rebsilva** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hey @rebsilva, can you please fix the source formatter issues and resend? Thanks

**Comment by rebsilva** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Pull request closed without merging**